popde/density_estimate.py

```diff
@@ -89,6 +88,9 @@ def __init__(self, data, weights=None, input_transf=None, stdize=False,
             #if self.weights.max() > 100. or self.weights.max() < 0.01:
             # Commented out as it's not clear if we ever need the 'unnormalized' case.
             self.normalize_weights()
+        else:
+            # Initialize uniform weights if not provided
+            self.kde_weights = None
```

Comment:
I don't see why this is needed but I guess it might be useful for `self.kde_weights` to exist, even if we aren't using weights.  But please remove the comment as it does not describe what the code does (it does not provide 'uniform weights').